get_value.py

```diff
@@ -7,14 +7,18 @@
 home = str(Path.home())
 config = load_config(f'{home}/.config/c20.yaml')
 num_tokens = config.get('num_tokens')
+currency = config.get('currency', 'USD')
+  
 
 # Get token value
 response = requests.get('https://us-central1-cryptodash1.cloudfunctions.net/fundValue')
 value_per_token = response.json().get('nav_per_token')
 
-# Get dollar to sek value
-response = requests.get('https://api.fixer.io/latest?base=USD')
-sek = response.json().get('rates').get('SEK')
-
-
-print(int(value_per_token*num_tokens*sek))
+if currency.upper() == 'USD':
+  exchange_rate = 1
+else:
+  # Get dollar to chosen currency exchange rate
+  response = requests.get('https://api.fixer.io/latest?base=USD')
+  exchange_rate = response.json().get('rates').get(currency)
```

Comment:
Do a currency.upper() here as well to match the one on line 17

Reply:
added .upper to line 10 instead.